auto_submit/lib/service/validation_service.dart

```diff
@@ -368,6 +372,41 @@ Exception: ${exception.message}
     return ProcessMergeResult.noMessage(true);
   }
 
+  /// A user can provide an alternate merge method via comment by providing the
+  /// string @autosubmit:<merge_method>.
+  static final RegExp regExpMergeMethod = RegExp(r'@autosubmit:(merge|squash|rebase)', caseSensitive: false);
+  Future<github.MergeMethod> checkForMergeMethod({
+    required github.RepositorySlug slug,
+    required int prNumber,
+  }) async {
+    const github.MergeMethod defaultMergeMethod = github.MergeMethod.squash;
+    final GithubService githubService = await config.createGithubService(slug);
+
+    final List<github.IssueComment> issueComments = await githubService.listIssueComments(slug, prNumber);
+    if (issueComments.isEmpty) {
+      return defaultMergeMethod;
+    }
+
+    final List<github.MergeMethod> foundMatches = [];
```

Comment:
nit: foundMatches -> foundMethod, since we are not storing the raw matched strings here, but the method that we are returning, and it matched defaultMethod as well.

Reply:
Gotcha. I can rename this for clarity.